about.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Digital Master Plan 2023</title>
+  <link rel="stylesheet" href="css/general.css">
+  <link rel="stylesheet" href="css/about.css">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+  </style>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+  <my-menu></my-menu>
+  <section class="section d-flex-column d-ds-flex-column justify-content-center align-items-center container" id="about">
+    <div>
+      <h1 class="title">
+        digital master plan global summit 2023
+      </h1>
+  
+      <p class="para">
+        The global summit brings together the community of experts, academics, and activist who comprise the Creative Commons affliate network in a different country every two, This year, we hope to expand our invitation list including organizations and individuals.
+      </p>
+  
+      <div>
+        <p class="txt-details">
+          please contact us per any Email for any Further questions
+          about Digital Master Plan<br>
+          <span class="address-info">digital@info.ac.ke</span>
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="section container d-flex-column align-items-center justify-content-center">
+    <h3 class="logo-title header-3 text-align-center">
+      digital master plan global summit 2023 Logo
+    </h3>
+    <p class="logo-text-info txt-details text-align-center">
+      The logo of Digital Master Plan 2023 was decided through the logo competition from 8, June to 7, July. After online public voting and Selection Committee's judging the process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we optimized the winning logo for this year's summit as following
+    </p>
+
+    <div class="logo-icon d-flex justify-content-center align-items-center">
+      <img class="dmp-logo" src="icons/Digital innovation_0.png" alt="logo">
+
+      <p class="logo-info">
+        Digital Master<br> Plan 2023
+      </p>
+    </div>
+  </section>
```

Comment:
- [ ] I noticed that you forgot to add the line after the **_Digital Master Plan Global Summit 2023 Logo_**. If you notice there is a very small in the design guidelines after this section's heading. Please make sure to add it.
- [ ] I think it would be better if you add a border after this section as given in the design guidelines and increase the `width` of the logo-icon div. What you think? 🤔🥇
- [ ] Also, please consider adding a little bit more padding/margin to the left and right side of the text and section's heading so the we have a design according to the design guidelines. I know you can do that 💪

### _Desktop version 🔽_
Your design | Design guidelines |
--- | --- | 
![image](https://user-images.githubusercontent.com/106119105/217344241-031e2b0c-4ce1-4e87-9432-fc7f38a56d19.png) | ![image](https://user-images.githubusercontent.com/106119105/217344482-ac421a58-65e3-494e-898b-8ec982a28161.png) | 

## **_Mobile version 🔽_**
Your design | Design guidelines |
--- | --- | 
![image](https://user-images.githubusercontent.com/106119105/217342993-be6c45aa-171d-4b67-804d-0849ee006dbd.png) | ![image](https://user-images.githubusercontent.com/106119105/217345240-2fbbe55d-21d8-47ef-8ddb-8282eaa18d4f.png) |